.github/workflows/build-operator-image.yaml

```diff
@@ -13,15 +13,16 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Build Image
-        run: |
-          IMG=eda-server-operator:main make docker-build
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
 
-      - name: Push To Quay
-        uses: redhat-actions/push-to-registry@v2.1.1
+      - name: Login to Quay.io
+        uses: docker/login-action@v1
         with:
-          image: eda-server-operator
-          tags: main
-          registry: quay.io/ansible/
+          registry: quay.io
           username: ${{ secrets.QUAY_USER }}
           password: ${{ secrets.QUAY_TOKEN }}
+
+      - name: Build and Push Image
+        run: |
+          docker buildx build --push --platform=linux/arm64,linux/amd64,linux/s390x,linux/ppc64le --tag quay.io/ansible/eda-server-operator:main -f Dockerfile .
```

Comment:
I took a very different approach to this file the reality is that there was only one line to change, one step to move, and one to add. none of the other steps are required.

``` diff
---

name: Build Operator Image

on:
  push:
    branches: [main]

jobs:
  release:
    runs-on: ubuntu-latest
    name: Push main image
    steps:
      - uses: actions/checkout@v2

+      - name: Log into registry quay.io
+        uses: docker/login-action@343f7c4344506bcbf9b4de18042ae17996df046d # v3.0.0
+         with:
+           registry: quay.io/ansible/
+           username: ${{ secrets.QUAY_USER }}
+           password: ${{ secrets.QUAY_TOKEN }}

      - name: Build Image
        run: |
-          IMG=eda-server-operator:main make docker-build
+         IMG=quay.io/ansible/eda-server-operator:main make docker-buildx

-      - name: Push To Quay
-        uses: redhat-actions/push-to-registry@v2.1.1
-        with:
-          image: eda-server-operator
-          tags: main
-          registry: quay.io/ansible/
-          username: ${{ secrets.QUAY_USER }}
-          password: ${{ secrets.QUAY_TOKEN }}

```

Docker buildx from the makefile also pushes, so no additional push steps are required. However prior to pushing, you must login.

Reply:
That makes me realize, I still had the full custom command in from an intermediary commit when I thought I needed to use `--load`, then  push, rather than just use the `--push` flag.  I just switched back to using the make target.

Thanks for the review!

Reply:
thats the beauty of the make file. can be used everywhere you can run a cli command